graylog-storage-elasticsearch7/src/test/java/org/graylog/storage/elasticsearch7/ElasticsearchFilterDeprecationWarningsInterceptorTest.java

```diff
@@ -0,0 +1,63 @@
+package org.graylog.storage.elasticsearch7;
+
+import org.graylog.shaded.elasticsearch7.org.apache.http.HttpException;
+import org.graylog.shaded.elasticsearch7.org.apache.http.HttpResponse;
+import org.graylog.shaded.elasticsearch7.org.apache.http.ProtocolVersion;
+import org.graylog.shaded.elasticsearch7.org.apache.http.message.BasicHttpResponse;
+import org.graylog.shaded.elasticsearch7.org.apache.http.message.BasicStatusLine;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class ElasticsearchFilterDeprecationWarningsInterceptorTest {
+    @Test
+    public void testInterceptorNoHeader() throws IOException, HttpException {
+        ElasticsearchFilterDeprecationWarningsInterceptor interceptor = new ElasticsearchFilterDeprecationWarningsInterceptor();
+
+        HttpResponse response = new BasicHttpResponse(new BasicStatusLine(new ProtocolVersion("HTTP", 0, 0), 0, null));
+        interceptor.process(response, null);
+
+        Assert.assertEquals( "Number of Headers should be 0 and the interceptor should not fail in itself.", 0, response.getAllHeaders().length);
```

Comment:
We prefer to use [AssertJ](https://assertj.github.io/doc/) for assertions, so this would look like this:
```suggestion
        assertThat(response.getAllHeaders())
                .as("Number of Headers should be 0 and the interceptor should not fail in itself.")
                .hasSize(0);
```

Reply:
fixed it like suggested